src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -24,29 +42,125 @@ import {CdkMenuItem} from './menu-item';
   exportAs: 'cdkMenuTriggerFor',
   host: {
     'aria-haspopup': 'menu',
+    '[attr.aria-expanded]': 'isSubmenuOpen()',
   },
 })
-export class CdkMenuItemTrigger implements AfterContentInit {
+export class CdkMenuItemTrigger implements OnDestroy {
   /** Template reference variable to the menu this trigger opens */
   @Input('cdkMenuTriggerFor') _menuPanel?: CdkMenuPanel;
 
+  /** Emits when the attached submenu is requested to open */
+  @Output() readonly cdkMenuOpened: EventEmitter<void> = new EventEmitter();
+
+  /** Emits when the attached submenu is requested to close  */
+  @Output() readonly cdkMenuClosed: EventEmitter<void> = new EventEmitter();
```

Comment:
Ah, I should have been more specific here; these can be
```
@Output('cdkMenuOpened') readonly opened: EventEmitter<void> = new EventEmitter();

@Output('cdkMenuClosed') readonly closed: EventEmitter<void> = new EventEmitter();
```
We always want the actual property in TypeScript to be unprefixed, with the prefix only aplpying to template bindings via the alias

Reply:
🤣  makes sense now.

Addressed